<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IonicWidget</class>
 <widget class="QWidget" name="IonicWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTableWidget" name="table_titration">
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>3</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Species</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Charge</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Initial</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Buret</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>Na</string>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>+1</string>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string>0.15</string>
      </property>
     </item>
     <item row="0" column="3">
      <property name="text">
       <string>0.1</string>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>Cl</string>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string>-1</string>
      </property>
     </item>
     <item row="1" column="2">
      <property name="text">
       <string>0.15</string>
      </property>
     </item>
     <item row="1" column="3">
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string>OH</string>
      </property>
     </item>
     <item row="2" column="1">
      <property name="text">
       <string>-1</string>
      </property>
     </item>
     <item row="2" column="2">
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item row="2" column="3">
      <property name="text">
       <string>0.1</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QGridLayout" name="gl_volume">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item row="0" column="1">
        <widget class="QLabel" name="lbl_Vi">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Initial</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="lbl_Vf">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Final</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lbl_V">
         <property name="text">
          <string>Volume</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="dsb_V0">
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
         <property name="value">
          <double>10.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QDoubleSpinBox" name="dsb_Vf">
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
         <property name="value">
          <double>11.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,10">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QPushButton" name="pb_plus">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>10</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_minus">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
